compiler/luci/pass/src/FuseAddWithTConvPass.cpp

```diff
@@ -90,9 +90,12 @@ bool fuse_add_with_tconv(luci::CircleTransposeConv *tconv)
 
   if (add->fusedActivationFunction() == luci::FusedActFunc::RELU6)
   {
+    auto name = addition->name();
+    assert(name.length() > 0);
```

Comment:
As mentioned https://github.com/Samsung/ONE/pull/6036#issuecomment-779575820 , if name is an optional field as TFlite, we should handle nodes with no name.

Reply:
I don't think we should follow like tflite. it's our format and our decision.